glide.yaml

```diff
@@ -15,7 +15,8 @@ import:
   - json
 - package: github.com/projectcalico/go-yaml-wrapper
 - package: github.com/projectcalico/libcalico-go
-  version: 97f275dffda5fcbad25248aec09b556173e09e92
+  repo: https://github.com/asincu/libcalico-go.git
```

Comment:
we probably don't want to merge this change into master :)

Reply:
@briansan  I have a PR opened on libcalico-go that needs to be merged before this one: https://github.com/projectcalico/libcalico-go/pull/1084